src/buttons/radio.ts

```diff
@@ -117,11 +132,17 @@ export class NgbRadio implements OnDestroy {
 
   get checked() { return this._checked; }
 
-  get disabled() { return this._disabled; }
+  get disabled() {
+    let result = (this._group && this._group.disabled) || this._disabled;
+    if (this._label) {
+      this._label.disabled = result;
```

Comment:
I didn't like that either. It is a bit similar to the previous comment. 
The question is how to keep the label in sync with the `input`. Again, I will have a look, but any suggestions appreciated.